### ISSUE TITLE
Fresh install of Node-RED on HA  0.92.2 (Hass.io); UI will not display

### ISSUE THREAD

**Issue by grantalewis** [hassio-addons/addon-node-red#217]
# Problem/Motivation

Home Assistant (Hass.io) v. 0.92.2  running on Virtual Box Ubuntu 64.

Was having problems with Node-RED (3.03) and so uninstalled, deleted the config/node-red folder, reinstalled, and then restored automations. Node-RED will not initialize. Cannot access the UI.

Node-RED Config:

```
{
  "credential_secret": "<redacted>",
  "dark_mode": false,
  "http_node": {
    "username": "",
    "password": ""
  },
  "http_static": {
    "username": "",
    "password": ""
  },
  "ssl": true,
  "certfile": "fullchain.pem",
  "keyfile": "privkey.pem",
  "require_ssl": true,
  "system_packages": [],
  "npm_packages": [],
  "init_commands": []
}
```

Hass.io System Logs:

```
19-05-05 18:39:19 INFO (MainThread) [__main__] Initialize Hass.io setup
19-05-05 18:39:19 INFO (MainThread) [__main__] Setup HassIO
19-05-05 18:39:19 INFO (SyncWorker_0) [hassio.docker.supervisor] Attach to Supervisor homeassistant/amd64-hassio-supervisor with version 162
19-05-05 18:39:19 INFO (MainThread) [hassio.utils.gdbus] Introspect org.freedesktop.systemd1 on /org/freedesktop/systemd1
19-05-05 18:39:19 INFO (MainThread) [hassio.utils.gdbus] Connect to dbus: org.freedesktop.systemd1 - /org/freedesktop/systemd1
19-05-05 18:39:19 INFO (MainThread) [hassio.utils.gdbus] Introspect org.freedesktop.hostname1 on /org/freedesktop/hostname1
19-05-05 18:39:19 INFO (MainThread) [hassio.utils.gdbus] Connect to dbus: org.freedesktop.hostname1 - /org/freedesktop/hostname1
19-05-05 18:39:19 INFO (MainThread) [hassio.utils.gdbus] Introspect de.pengutronix.rauc on /
19-05-05 18:39:19 INFO (MainThread) [hassio.utils.gdbus] Connect to dbus: de.pengutronix.rauc - /
19-05-05 18:39:19 INFO (MainThread) [hassio.host.info] Update local host information
19-05-05 18:39:19 INFO (MainThread) [hassio.utils.gdbus] Call org.freedesktop.DBus.Properties.GetAll on /org/freedesktop/hostname1
19-05-05 18:39:19 INFO (MainThread) [hassio.host.services] Update service information
19-05-05 18:39:19 INFO (MainThread) [hassio.utils.gdbus] Call org.freedesktop.systemd1.Manager.ListUnits on /org/freedesktop/systemd1
19-05-05 18:39:19 INFO (MainThread) [hassio.host.apparmor] Load AppArmor Profiles: {'hassio-supervisor'}
19-05-05 18:39:19 INFO (MainThread) [hassio.host.services] Reload local service hassos-apparmor.service
19-05-05 18:39:19 INFO (MainThread) [hassio.utils.gdbus] Call org.freedesktop.systemd1.Manager.ReloadOrRestartUnit on /org/freedesktop/systemd1
19-05-05 18:39:20 INFO (SyncWorker_1) [hassio.docker.interface] Attach to homeassistant/qemux86-64-homeassistant with version 0.92.2
19-05-05 18:39:20 INFO (MainThread) [hassio.hassos] Detect HassOS 2.11 on host system
19-05-05 18:39:20 INFO (SyncWorker_3) [hassio.docker.hassos_cli] Found HassOS CLI homeassistant/amd64-hassio-cli with version 9
19-05-05 18:39:20 INFO (MainThread) [hassio.addons.git] Load add-on /data/addons/core repository
19-05-05 18:39:20 INFO (MainThread) [hassio.addons.git] Load add-on /data/addons/git/03d23610 repository
19-05-05 18:39:20 INFO (MainThread) [hassio.addons.git] Load add-on /data/addons/git/a0d7b954 repository
19-05-05 18:39:20 INFO (MainThread) [hassio.addons] Load add-ons: 56 all - 56 new - 0 remove
19-05-05 18:39:20 INFO (SyncWorker_4) [hassio.docker.interface] Attach to hassioaddons/node-red-amd64 with version v3.0.1
19-05-05 18:39:20 INFO (SyncWorker_3) [hassio.docker.interface] Attach to homeassistant/amd64-addon-duckdns with version 1.6
19-05-05 18:39:20 INFO (SyncWorker_5) [hassio.docker.interface] Attach to homeassistant/amd64-addon-samba with version 8.1
19-05-05 18:39:20 INFO (SyncWorker_0) [hassio.docker.interface] Attach to homeassistant/amd64-addon-mosquitto with version 4.1
19-05-05 18:39:20 INFO (SyncWorker_6) [hassio.docker.interface] Attach to homeassistant/amd64-addon-ssh with version 5.4
19-05-05 18:39:20 INFO (SyncWorker_4) [hassio.docker.interface] Attach to danimtb/amd64-dasshio with version 0.3.6
19-05-05 18:39:20 INFO (MainThread) [hassio.updater] Fetch update data from https://s3.amazonaws.com/hassio-version/stable.json
19-05-05 18:39:20 INFO (MainThread) [hassio.snapshots] Found 1 snapshot files
19-05-05 18:39:20 INFO (MainThread) [hassio.discovery] Load 1 messages
19-05-05 18:39:20 INFO (MainThread) [hassio.ingress] Load 14 ingress session
19-05-05 18:39:20 INFO (MainThread) [__main__] Run Hass.io
19-05-05 18:39:20 INFO (MainThread) [hassio.misc.dns] Start DNS port forwarding for host add-ons
19-05-05 18:39:20 INFO (MainThread) [hassio.api] Start API on 172.30.32.2
19-05-05 18:39:20 INFO (MainThread) [hassio.addons] Startup initialize run 0 add-ons
19-05-05 18:39:20 INFO (MainThread) [hassio.addons] Startup system run 1 add-ons
19-05-05 18:39:21 INFO (SyncWorker_3) [hassio.docker.addon] Start Docker add-on homeassistant/amd64-addon-mosquitto with version 4.1
19-05-05 18:39:21 INFO (MainThread) [hassio.services.modules.mqtt] Set core_mosquitto as service provider for mqtt
19-05-05 18:39:21 INFO (MainThread) [hassio.discovery] Duplicate discovery message from core_mosquitto
19-05-05 18:39:26 INFO (MainThread) [hassio.addons] Startup services run 4 add-ons
19-05-05 18:39:26 INFO (SyncWorker_2) [hassio.docker.addon] Start Docker add-on homeassistant/amd64-addon-samba with version 8.1
19-05-05 18:39:27 INFO (SyncWorker_3) [hassio.docker.addon] Start Docker add-on danimtb/amd64-dasshio with version 0.3.6
19-05-05 18:39:27 INFO (SyncWorker_1) [hassio.docker.addon] Start Docker add-on homeassistant/amd64-addon-duckdns with version 1.6
19-05-05 18:39:27 INFO (SyncWorker_8) [hassio.docker.addon] Start Docker add-on homeassistant/amd64-addon-ssh with version 5.4
19-05-05 18:39:32 INFO (SyncWorker_7) [hassio.docker.interface] Start homeassistant/qemux86-64-homeassistant
19-05-05 17:25:13 INFO (MainThread) [hassio.homeassistant] Detect a running Home Assistant instance
19-05-05 17:25:13 INFO (MainThread) [hassio.addons] Startup application run 1 add-ons
19-05-05 17:25:14 INFO (SyncWorker_8) [hassio.docker.addon] Start Docker add-on hassioaddons/node-red-amd64 with version v3.0.1
19-05-05 17:25:15 INFO (MainThread) [hassio.api.security] /host/info access from a0d7b954_nodered
19-05-05 17:25:19 INFO (MainThread) [hassio.tasks] All core tasks are scheduled
19-05-05 17:25:19 INFO (MainThread) [hassio.core] Hass.io is up and running
19-05-05 17:25:33 INFO (MainThread) [hassio.api.proxy] Home Assistant WebSocket API request initialize
19-05-05 17:25:33 INFO (MainThread) [hassio.api.proxy] WebSocket access from a0d7b954_nodered
19-05-05 17:25:33 INFO (MainThread) [hassio.homeassistant] Updated Home Assistant API token
19-05-05 17:25:33 INFO (MainThread) [hassio.api.proxy] Home Assistant WebSocket API request running
```

Node-RED Logs:

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Hass.io Add-on: Node-RED
 Flow-based programming for the Internet of Things
-----------------------------------------------------------
 Add-on version: 3.0.1
 You are running the latest version of this add-on.
 System: HassOS 2.11  (amd64 / qemux86-64)
 Home Assistant version: 0.92.2
 Supervisor version: 162
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] nginx.sh: executing... 
[cont-init.d] nginx.sh: exited 0.
[cont-init.d] node-red.sh: executing... 
audited 1 package in 1.159s
found 0 vulnerabilities
[cont-init.d] node-red.sh: exited 0.
[cont-init.d] user.sh: executing... 
[cont-init.d] user.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[13:25:20] INFO: Starting Node-RED...
> hassio-addon-node-red@ start /opt
> node $NODE_OPTIONS node_modules/node-red/red.js "--settings" "/etc/node-red/config.js"
5 May 13:25:23 - [info] 
Welcome to Node-RED
===================
5 May 13:25:23 - [info] Node-RED version: v0.20.5
5 May 13:25:23 - [info] Node.js  version: v10.14.2
5 May 13:25:23 - [info] Linux 4.19.20 x64 LE
5 May 13:25:23 - [info] Loading palette nodes
5 May 13:25:27 - [warn] rpi-gpio : Raspberry Pi specific node set inactive
5 May 13:25:28 - [info] Settings file  : /etc/node-red/config.js
5 May 13:25:28 - [info] Context store  : 'default' [module=memory]
5 May 13:25:28 - [info] User directory : /config/node-red/
5 May 13:25:28 - [warn] Projects disabled : editorTheme.projects.enabled=false
5 May 13:25:28 - [info] Flows file     : /config/node-red/flows.json
5 May 13:25:28 - [info] Server now running at http://127.0.0.1:46836/
5 May 13:25:28 - [info] Starting flows
5 May 13:25:28 - [info] Started flows
[13:25:28] INFO: Starting NGinx...
5 May 13:25:30 - [warn] [api-current-state:Temp] entity could not be found in cache for entity_id: sensor.dark_sky_temperature, sending empty payload
5 May 13:25:30 - [warn] [api-current-state:FR shades down?] entity could not be found in cache for entity_id: switch.family_room_fr_shades_indicator, sending empty payload
5 May 13:25:33 - [info] [server:Home Assistant] WebSocket Connected to http://hassio/homeassistant
5 May 13:25:48 - [warn] [api-current-state:Office Light Sensor off?] entity could not be found in cache for entity_id: switch.officelux, sending empty payload
5 May 13:26:48 - [warn] [api-current-state:Office Light Sensor off?] entity could not be found in cache for entity_id: switch.officelux, sending empty payload
5 May 13:27:24 - [warn] [api-current-state:Coffee Pot On?] entity could not be found in cache for entity_id: switch.mqtt_sonoff_coffee, sending empty payload
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET / HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /vendor/bootstrap/css/bootstrap.min.css HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /vendor/jquery/css/smoothness/jquery-ui-1.10.3.custom.min.css HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /vendor/font-awesome/css/font-awesome.min.css HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /red/images/node-red.png HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /red/images/spin.svg HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /vendor/vendor.css HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /red/style.min.css HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /vendor/font-awesome/fonts/fontawesome-webfont.woff2?v=4.7.0 HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /vendor/ace/ext-language_tools.js HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /vendor/ace/ace.js HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /vendor/jsonata/jsonata.min.js HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /red/red.min.js HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /vendor/vendor.js HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:47 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /red/main.min.js HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
[05/May/2019:13:27:48 -0400] 304 71.81.123.246, 172.30.32.1(172.30.32.2) GET /red/images/grip.png HTTP/1.1 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36)
5 May 13:27:48 - [warn] [api-current-state:Office Light Sensor off?] entity could not be found in cache for entity_id: switch.officelux, sending empty payload
```

## Expected behavior

Working add-on / access to the UI.

## Actual behavior

Can't access UI.

**Comment by grantalewis** [comment]
Following up to report that I am able to access the UI... eventually, after a period of time. It's like whatever process responsible for rendering the graphical elements -- for instance, the images to the left of the individual nodes (events: state, trigger: state, etc.) -- is having trouble rendering those images. 

Here's an example of the issue involving the Deploy menu:
https://i.imgur.com/rDiSoWS.png

It's almost as if there's a bandwidth issue. If I wait long enough, click around enough, the problem eventually resolves and I see the whole UI as intended. If I restart, the problem returns.

**Comment by frenck** [comment]
You are running add-on version 3.0.1, this issue was addressed in 3.0.2. Please upgrade.
